webviewer/src/pages/Account/components/Sidebar/Sidebar.jsx

```diff
@@ -12,24 +14,29 @@ const breadcrumbsItems = [
 
 export const Sidebar = props => {
   const { activeTab, setActiveTab } = props;
+  const user = useSelector(selectUser);
   const history = useHistory();
 
   return (
     <StyledSidebar>
       <StyledBreadcrumbs items={breadcrumbsItems} />
       <div>
-        {TABS.map(({ name, icon, hash }) => (
-          <TabItem
-            key={name}
-            isActive={name === activeTab}
-            onClick={() => {
-              setActiveTab(name);
-              history.push({ hash });
-            }}
-          >
-            <Icon>{icon}</Icon> {name}
-          </TabItem>
-        ))}
+        {TABS.map(({ name, icon, hash }) => {
+          if (!user.trial_finished_at && name === TAB_NAMES.transactions) return;
```

Comment:
not best practice to use .map for skipping elements. maybe better to use some pre-processing before rendering?